### PR TITLE
Refactor management of scenarios

### LINKED ISSUE #50: Find a solution for formatting WKT on frontend
We're passing geometries from client to server in WKT, but right now we're doing the string formatting manually. There might be a library that makes that easier.

### LINKED ISSUE #75: parcel coordinates are incorrect when parcel spans tiles
When we click on a feature from a `VectorTile` layer (i.e. a parcel) and get it's `flatCoordinates`, these coordinates only represent the piece of that feature within the tile that was clicked.

See the image where the clicked parcel extends off the map and is highlighted in darker blue, while the coordinates that we extracted and sent to the server as WKT to represent this parcel are in the lighter blue.

)

So depending where in the parcel I click, the coordinates we use can vary.

The good news is, we already have the logic to find all the "pieces" of the parcel that span multiple tiles and add them to a single selection layer -- that's how we can correctly highlight the entire parcel (dark blue). So presumably we can extract the coordinates from that layer instead of from the individual feature that was clicked.

### PR DESCRIPTION
**Pull request by davemfish**
This is mostly frontend updates to the way scenario data is managed in react. Also some linking of study areas & scenario data to the map.

On the backend, there is new `JOBTYPE` for simply cropping an LULC, as a means to get the data representing a "baseline" scenario. 

Fixes #75 
Fixes #50 

@dcdenu4 , like the last one, I'm very much okay if you only want to look at the backend changes here.

### CODE REVIEW

File: server/sql_app/main.py [hunk 1]
```diff
@@ -155,8 +156,13 @@ def create_study_area(
     db_session = crud.get_session(db, session_id=session_id)
     if db_session is None:
         raise HTTPException(status_code=404, detail="Session not found")
-    return crud.create_study_area(
+    db_study_area = crud.create_study_area(
         db=db, **new_area.dict(), session_id=session_id)
+    # scenario = schemas.ScenarioBase
+    # scenario.name = 'baseline'
+    # _ = crud.create_scenario(
+    #     db=db, scenario=scenario, study_area_id=db_study_area.id)
```
Comment (dcdenu4) [inline server/sql_app/main.py:164]: Should this be commented out? Is the frontend handling the baseline creation?

File: server/sql_app/main.py [hunk 2]
```diff
@@ -187,14 +193,27 @@ def update_study_area(session_id: str, study_area: schemas.StudyArea,
 @app.get("/study_areas/{session_id}", response_model=list[schemas.StudyArea])
 def get_study_areas(session_id: str, db: Session = Depends(get_db)):
     # check that the session exists
+    # TODO: when & why do we need to get the session?
```
Comment (dcdenu4) [inline server/sql_app/main.py:196]: I *think* I had general thoughts of, should a user have to verify their session as an authentication step before requesting other data.

File: server/sql_app/main.py [hunk 3]
```diff
@@ -187,14 +193,27 @@ def update_study_area(session_id: str, study_area: schemas.StudyArea,
...
+@app.get("/scenario/{study_area_id}", response_model=list[schemas.Scenario])
+def get_scenarios(study_area_id: int, db: Session = Depends(get_db)):
+    db_scenarios = crud.get_scenarios(db=db, study_area_id=study_area_id)
+    return db_scenarios
+
+
+@app.get("/scenario/{scenario_id}", response_model=schemas.Scenario)
+def get_scenario(scenario_id: int, db: Session = Depends(get_db)):
+    db_scenarios = crud.get_scenario(db=db, scenario_id=scenario_id)
+    return db_scenarios
```
Comment (dcdenu4) [inline server/sql_app/main.py:213]:
```suggestion
    db_scenario = crud.get_scenario(db=db, scenario_id=scenario_id)
    return db_scenario
```

**Review by dcdenu4 — CHANGES_REQUESTED** [review]
Hey @davemfish , this looks good. Just a trivial change request and comment.

File: server/sql_app/main.py (hunk 2 above)
Comment (davemfish) [reply server/sql_app/main.py:196]: Ah, that's helpful, thanks. I'm not sure I've been consistent with including that or not.

File: server/sql_app/main.py (hunk 1 above)
Comment (davemfish) [reply server/sql_app/main.py]: Thanks, I had an unpushed commit that remove removed some commented code like this. Just pushed.

File: server/sql_app/main.py (hunk 3 above)
Comment (davemfish) [reply server/sql_app/main.py]: Changed

**Review by dcdenu4 — APPROVED** [review]
Looks good!

**Pull request merged**